docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -0,0 +1,38 @@
+=================================
+Contributing a "What's New" entry
+=================================
+
+Iris has an aggregator for building an initial "What's New" document for each release, from sections contributed by code authors. This enables the release documentation to be written by the developer most familiar with the change while permitting editing to take place.
+
+Each release should have a directory, "contributions_<versionnumber>". Into this directory can be placed items for inclusion into the release documentation. The filename should be structured as follows:
+
+<category>_<date>_<summary>.txt
+
+The category will be one of the following:
+
+bugfix
+  A bugfix.
+newfeature
+  Features that are new or changed to add functionality.
+incompatiblechange
+  A change that causes an incompatibility with prior versions of Iris.
+deprecate
+  Deprecations of functionality.
+docchange
+  Changes to documentation.
+
```

Comment:
They're defined as a global list of dicts. The rationale being that allowing self-definition of categories leads to inconsistency.
(Lines 32:38).